src/library_fetchfs.js

```diff
@@ -5,10 +5,19 @@
  */
 
 addToLibrary({
-  $FETCHFS__deps: ['$stringToUTF8OnStack', 'wasmfs_create_fetch_backend'],
+  $FETCHFS__deps: ['$stringToUTF8OnStack', 'wasmfs_create_fetch_backend', 'wasmfs_fetch_create_manifest', 'wasmfs_fetch_add_to_manifest'],
   $FETCHFS: {
     createBackend(opts) {
-      return _wasmfs_create_fetch_backend(stringToUTF8OnStack(opts.base_url));
+      var manifest = 0;
+      if (opts['manifest']) {
+        manifest = wasmfs_fetch_create_manifest();
+        Object.entries(opts['manifest']).forEach((pair) => {
```

Comment:
Can you do `([key, value])` or `([path, url])` here?

Reply:
Good idea.